src/components/Menu/index.js

```diff
@@ -0,0 +1,158 @@
+import { Grid, Button, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Link from "@/pesayetu/components/Link";
+import SocialMediaIcons from "@/pesayetu/components/SocialMediaIcons";
+
+const useStyles = makeStyles(({ typography, breakpoints, palette }) => ({
+  root: {
+    margin: `${typography.pxToRem(20)} 0`,
```

Comment:
We should also revise this a bit, to make sure we have the correct total height/ or close enough to the design